pkg/reconciler/internal/values/values.go

```diff
@@ -32,6 +32,14 @@ type Values struct {
 }
 
 func FromUnstructured(obj *unstructured.Unstructured) (*Values, error) {
```

Comment:
I guess I was trying to avoid the back-and-forth of `New`->`Map`->`New`, not for performance reasons but for simplicity. But getting rid of `ChartValuesFromUnstructured` seems worth it. Changed.